acceptance/suites/pre_suite/foss/00_setup_environment.rb

```diff
@@ -1,2 +1,13 @@
-step "Initialize Test Config"
+step "Initialize Test Config" do
   PuppetServerExtensions.initialize_config options
+
+  if PuppetServerExtensions.config[:puppet_build_version] == 'LATEST'
```

Comment:
What do we want our default behavior to be? If we expect our acceptance tests to default to running against the latest version we can remove the default values and use the latest on empty values. I'm not strongly opinionated either way.